auditapi/go.mod

```diff
@@ -1,12 +1,7 @@
 module github.com/phoenixnap/go-sdk-bmc/auditapi
 
-go 1.17
-
-require golang.org/x/oauth2 v0.0.0-20210218202405-ba52d332ba99
+go 1.13
```

Comment:
I believe that the generator has overwritten the go.mod file for us here. Typically we used to manually maintain mod files in this repo, and used to ask generator to never generate them.
At this point, shall we maybe modify the template to generate correct go.mod files for us?

Reply:
Updated to use version `1.17` - I feel like using templates would be better to avoid more files that we need to manually maintain.

Reply:
I totally agree 👏